src/anyOf.ts

```diff
@@ -0,0 +1,3 @@
+export default function anyOf(input: string) {
```

Comment:
This deviates a bit from the old API, but I feel renaming this function to `anyCharacterFrom` would make the API more intuitive.